### PR TITLE
Add grayskull

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Adding ``grayskull``, recipe generated with [grayskull](https://github.com/marcelotrevisani/grayskull/) :skull: 

Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@conda-forge/staged-recipes please review :)

File: recipes/grayskull/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 654485aae458db4565eee55beaae565bd1417dee266faace458f4fce0f750ad2
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - grayskull = grayskull.__main__:main
+    - gs = grayskull.__main__:main
```
Comment (ocefpaf) [inline recipes/grayskull/meta.yaml]: As a ghostscript user I'm not really ok with the `gs` short version. However, I understand that even the long `grayskull` command is a temporary name, right?

Comment (marcelotrevisani) [reply recipes/grayskull/meta.yaml]: Yes, but I can remove it if you want to

**Comment by jakirkham** [comment]
Woah! This is exciting! Does this mean it is ready for broader usage? 😄

**Pull request merged**